[Bug]: unable to load script spreadsheetimporter error in BAS with UI5 version 1.120.13

### OData Version

OData V4

### Draft

Yes

### Scenario

Fiori Element

### Environment

BAS

### UI5 Spreadsheet Component

1.0.0

### What happened?

I am trying to install UI5 spreadsheet component in BAS, followed steps mentioned in the Getting Started guide. In custom button creation of component is failing with error: failed to load 'cc/spreadsheetimporter/v1_0_0/Component.js' from sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component.js: 404 not found.

Please see the attached document for details steps and screenshots.
[Excel upload configurations steps.docx](https://github.com/user-attachments/files/15742187/Excel.upload.configurations.steps.docx)


### Relevant log output

<details>

```shell
fetch-dbg.js:209 
        
        
       GET https://port4004-workspaces-ws-v9gxb.eu12.int.applicationstudio.cloud.sap/sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/manifest.json?sap-language=EN 404 (Not Found)
(anonymous) @ fetch-dbg.js:209
n @ fetch-dbg.js:147
t @ mixedFetch-dbg.js:55
i.loadResource @ LoaderExtensions-dbg.js:320
_.load @ Manifest-dbg.js:816
V @ Component-dbg.js:3103
z @ Component-dbg.js:2739
j.create @ Component-dbg.js:2455
n @ Component-dbg.js:1507
b.runWithOwner @ ManagedObject-dbg.js:1213
j.runAsOwner @ Component-dbg.js:762
j._createComponent @ Component-dbg.js:1518
j.createComponent @ Component-dbg.js:1338
Excelupload @ Excelupload.js?eval:11
eval @ FPMHelper.ts:48
(anonymous) @ ui5loader-dbg.js:2107
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2106
Oe @ ui5loader-dbg.js:1858
r @ ui5loader-dbg.js:2099
eval @ FPMHelper.ts:45
actionWrapper @ FPMHelper.ts:15
y @ ExpressionParser-dbg.js:412
(anonymous) @ ExpressionParser-dbg.js:735
l @ ExpressionParser-dbg.js:915
l.getExternalValue @ CompositeBinding-dbg.js:316
u @ EventHandlerResolver-dbg.js:349
eval @ EventHandlerResolver-dbg.js:173
r.fireEvent @ EventProvider-dbg.js:241
y.fireEvent @ Element-dbg.js:683
(anonymous) @ ManagedObjectMetadata-dbg.js:826
S.ontap @ library-preload.js?eval:206
y._handleEvent @ Element-dbg.js:352
N._handleEvent @ UIArea-dbg.js:1054
dispatch @ jquery-dbg.js:5430
c @ jquery-mobile-custom-dbg.js:1907
d @ jquery-mobile-custom-dbg.js:2030
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
trigger @ jquery-dbg.js:8823
(anonymous) @ jquery-dbg.js:8901
each @ jquery-dbg.js:385
each @ jquery-dbg.js:207
trigger @ jquery-dbg.js:8900
j @ jquery-mobile-custom-dbg.js:1455
L @ jquery-mobile-custom-dbg.js:1465
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
ui5loader-dbg.js:1463 
        
        
       GET https://port4004-workspaces-ws-v9gxb.eu12.int.applicationstudio.cloud.sap/sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component-preload.js net::ERR_ABORTED 404 (Not Found)
Ae @ ui5loader-dbg.js:1463
qe @ ui5loader-dbg.js:1681
Be @ ui5loader-dbg.js:2346
T @ Component-dbg.js:3232
(anonymous) @ Component-dbg.js:3449
Promise.then (async)
V @ Component-dbg.js:3440
z @ Component-dbg.js:2739
j.create @ Component-dbg.js:2455
n @ Component-dbg.js:1507
b.runWithOwner @ ManagedObject-dbg.js:1213
j.runAsOwner @ Component-dbg.js:762
j._createComponent @ Component-dbg.js:1518
j.createComponent @ Component-dbg.js:1338
Excelupload @ Excelupload.js?eval:11
eval @ FPMHelper.ts:48
(anonymous) @ ui5loader-dbg.js:2107
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2106
Oe @ ui5loader-dbg.js:1858
r @ ui5loader-dbg.js:2099
eval @ FPMHelper.ts:45
actionWrapper @ FPMHelper.ts:15
y @ ExpressionParser-dbg.js:412
(anonymous) @ ExpressionParser-dbg.js:735
l @ ExpressionParser-dbg.js:915
l.getExternalValue @ CompositeBinding-dbg.js:316
u @ EventHandlerResolver-dbg.js:349
eval @ EventHandlerResolver-dbg.js:173
r.fireEvent @ EventProvider-dbg.js:241
y.fireEvent @ Element-dbg.js:683
(anonymous) @ ManagedObjectMetadata-dbg.js:826
S.ontap @ library-preload.js?eval:206
y._handleEvent @ Element-dbg.js:352
N._handleEvent @ UIArea-dbg.js:1054
dispatch @ jquery-dbg.js:5430
c @ jquery-mobile-custom-dbg.js:1907
d @ jquery-mobile-custom-dbg.js:2030
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
trigger @ jquery-dbg.js:8823
(anonymous) @ jquery-dbg.js:8901
each @ jquery-dbg.js:385
each @ jquery-dbg.js:207
trigger @ jquery-dbg.js:8900
j @ jquery-mobile-custom-dbg.js:1455
L @ jquery-mobile-custom-dbg.js:1465
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
launchpadPage.html#sesmassuploadappsesappwithexce-display&/SESMainHeader(ID=8aae3bfa-a7bb-4883-b66d-344c8d62bd4b,IsActiveEntity=false)?layout=TwoColumnsMidExpanded:1 Refused to execute script from 'https://port4004-workspaces-ws-v9gxb.eu12.int.applicationstudio.cloud.sap/sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component-preload.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
ui5loader-dbg.js:1463 
        
        
       GET https://port4004-workspaces-ws-v9gxb.eu12.int.applicationstudio.cloud.sap/sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component-preload.js net::ERR_ABORTED 404 (Not Found)
Ae @ ui5loader-dbg.js:1463
i @ ui5loader-dbg.js:1448
error (async)
Ae @ ui5loader-dbg.js:1461
qe @ ui5loader-dbg.js:1681
Be @ ui5loader-dbg.js:2346
T @ Component-dbg.js:3232
(anonymous) @ Component-dbg.js:3449
Promise.then (async)
V @ Component-dbg.js:3440
z @ Component-dbg.js:2739
j.create @ Component-dbg.js:2455
n @ Component-dbg.js:1507
b.runWithOwner @ ManagedObject-dbg.js:1213
j.runAsOwner @ Component-dbg.js:762
j._createComponent @ Component-dbg.js:1518
j.createComponent @ Component-dbg.js:1338
Excelupload @ Excelupload.js?eval:11
eval @ FPMHelper.ts:48
(anonymous) @ ui5loader-dbg.js:2107
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2106
Oe @ ui5loader-dbg.js:1858
r @ ui5loader-dbg.js:2099
eval @ FPMHelper.ts:45
actionWrapper @ FPMHelper.ts:15
y @ ExpressionParser-dbg.js:412
(anonymous) @ ExpressionParser-dbg.js:735
l @ ExpressionParser-dbg.js:915
l.getExternalValue @ CompositeBinding-dbg.js:316
u @ EventHandlerResolver-dbg.js:349
eval @ EventHandlerResolver-dbg.js:173
r.fireEvent @ EventProvider-dbg.js:241
y.fireEvent @ Element-dbg.js:683
(anonymous) @ ManagedObjectMetadata-dbg.js:826
S.ontap @ library-preload.js?eval:206
y._handleEvent @ Element-dbg.js:352
N._handleEvent @ UIArea-dbg.js:1054
dispatch @ jquery-dbg.js:5430
c @ jquery-mobile-custom-dbg.js:1907
d @ jquery-mobile-custom-dbg.js:2030
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
trigger @ jquery-dbg.js:8823
(anonymous) @ jquery-dbg.js:8901
each @ jquery-dbg.js:385
each @ jquery-dbg.js:207
trigger @ jquery-dbg.js:8900
j @ jquery-mobile-custom-dbg.js:1455
L @ jquery-mobile-custom-dbg.js:1465
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
launchpadPage.html#sesmassuploadappsesappwithexce-display&/SESMainHeader(ID=8aae3bfa-a7bb-4883-b66d-344c8d62bd4b,IsActiveEntity=false)?layout=TwoColumnsMidExpanded:1 Refused to execute script from 'https://port4004-workspaces-ws-v9gxb.eu12.int.applicationstudio.cloud.sap/sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component-preload.js' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
Log-dbg.js:499 2024-06-07 11:07:14.010300 failed to load JavaScript resource: cc/spreadsheetimporter/v1_0_0/Component-preload.js -  sap.ui.ModuleSystem
g @ Log-dbg.js:499
t.error @ Log-dbg.js:249
L.error @ Log-dbg.js:685
i @ ui5loader-dbg.js:1452
error (async)
Ae @ ui5loader-dbg.js:1461
i @ ui5loader-dbg.js:1448
error (async)
Ae @ ui5loader-dbg.js:1461
qe @ ui5loader-dbg.js:1681
Be @ ui5loader-dbg.js:2346
T @ Component-dbg.js:3232
(anonymous) @ Component-dbg.js:3449
Promise.then (async)
V @ Component-dbg.js:3440
z @ Component-dbg.js:2739
j.create @ Component-dbg.js:2455
n @ Component-dbg.js:1507
b.runWithOwner @ ManagedObject-dbg.js:1213
j.runAsOwner @ Component-dbg.js:762
j._createComponent @ Component-dbg.js:1518
j.createComponent @ Component-dbg.js:1338
Excelupload @ Excelupload.js?eval:11
eval @ FPMHelper.ts:48
(anonymous) @ ui5loader-dbg.js:2107
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2106
Oe @ ui5loader-dbg.js:1858
r @ ui5loader-dbg.js:2099
eval @ FPMHelper.ts:45
actionWrapper @ FPMHelper.ts:15
y @ ExpressionParser-dbg.js:412
(anonymous) @ ExpressionParser-dbg.js:735
l @ ExpressionParser-dbg.js:915
l.getExternalValue @ CompositeBinding-dbg.js:316
u @ EventHandlerResolver-dbg.js:349
eval @ EventHandlerResolver-dbg.js:173
r.fireEvent @ EventProvider-dbg.js:241
y.fireEvent @ Element-dbg.js:683
(anonymous) @ ManagedObjectMetadata-dbg.js:826
S.ontap @ library-preload.js?eval:206
y._handleEvent @ Element-dbg.js:352
N._handleEvent @ UIArea-dbg.js:1054
dispatch @ jquery-dbg.js:5430
c @ jquery-mobile-custom-dbg.js:1907
d @ jquery-mobile-custom-dbg.js:2030
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
trigger @ jquery-dbg.js:8823
(anonymous) @ jquery-dbg.js:8901
each @ jquery-dbg.js:385
each @ jquery-dbg.js:207
trigger @ jquery-dbg.js:8900
j @ jquery-mobile-custom-dbg.js:1455
L @ jquery-mobile-custom-dbg.js:1465
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
ui5loader-dbg.js:1392 
        
        
       GET https://port4004-workspaces-ws-v9gxb.eu12.int.applicationstudio.cloud.sap/sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component.js 404 (Not Found)
je @ ui5loader-dbg.js:1392
qe @ ui5loader-dbg.js:1650
Oe @ ui5loader-dbg.js:1843
r @ ui5loader-dbg.js:2099
(anonymous) @ Component-dbg.js:3632
(anonymous) @ Component-dbg.js:3630
Promise.then (async)
V @ Component-dbg.js:3625
z @ Component-dbg.js:2739
j.create @ Component-dbg.js:2455
n @ Component-dbg.js:1507
b.runWithOwner @ ManagedObject-dbg.js:1213
j.runAsOwner @ Component-dbg.js:762
j._createComponent @ Component-dbg.js:1518
j.createComponent @ Component-dbg.js:1338
Excelupload @ Excelupload.js?eval:11
eval @ FPMHelper.ts:48
(anonymous) @ ui5loader-dbg.js:2107
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2106
Oe @ ui5loader-dbg.js:1858
r @ ui5loader-dbg.js:2099
eval @ FPMHelper.ts:45
actionWrapper @ FPMHelper.ts:15
y @ ExpressionParser-dbg.js:412
(anonymous) @ ExpressionParser-dbg.js:735
l @ ExpressionParser-dbg.js:915
l.getExternalValue @ CompositeBinding-dbg.js:316
u @ EventHandlerResolver-dbg.js:349
eval @ EventHandlerResolver-dbg.js:173
r.fireEvent @ EventProvider-dbg.js:241
y.fireEvent @ Element-dbg.js:683
(anonymous) @ ManagedObjectMetadata-dbg.js:826
S.ontap @ library-preload.js?eval:206
y._handleEvent @ Element-dbg.js:352
N._handleEvent @ UIArea-dbg.js:1054
dispatch @ jquery-dbg.js:5430
c @ jquery-mobile-custom-dbg.js:1907
d @ jquery-mobile-custom-dbg.js:2030
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
trigger @ jquery-dbg.js:8823
(anonymous) @ jquery-dbg.js:8901
each @ jquery-dbg.js:385
each @ jquery-dbg.js:207
trigger @ jquery-dbg.js:8900
j @ jquery-mobile-custom-dbg.js:1455
L @ jquery-mobile-custom-dbg.js:1465
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
ui5loader-dbg.js:1156 Uncaught (in promise) ModuleError: failed to load 'cc/spreadsheetimporter/v1_0_0/Component.js' from sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component.js: 404
    at we (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:6645)
    at ge.failWith (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:4709)
    at qe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:13312)
    at Oe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:15314)
    at Object.r [as require] (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:18696)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36487
    at new Promise (<anonymous>)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36454
Caused by: XHRLoadError: 404
    at n (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:9251)
    at XMLHttpRequest.<anonymous> (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:9504)
    at je (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:9596)
    at qe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:13287)
    at Oe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:15314)
    at Object.r [as require] (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:18696)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36487
    at new Promise (<anonymous>)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36454Caused by: XHRLoadError: 404
    at n (ui5loader-dbg.js:1369:12)
    at XMLHttpRequest.<anonymous> (ui5loader-dbg.js:1382:21)
    at je (ui5loader-dbg.js:1392:8)
    at qe (ui5loader-dbg.js:1650:5)
    at Oe (ui5loader-dbg.js:1843:20)
    at Object.r [as require] (ui5loader-dbg.js:2099:4)
    at Component-dbg.js:3632:13
    at new Promise (<anonymous>)
    at Component-dbg.js:3630:12
we @ ui5loader-dbg.js:1156
failWith @ ui5loader-dbg.js:934
qe @ ui5loader-dbg.js:1655
Oe @ ui5loader-dbg.js:1843
r @ ui5loader-dbg.js:2099
(anonymous) @ Component-dbg.js:3632
(anonymous) @ Component-dbg.js:3630
Promise.then (async)
j.registerForDestroy @ Component-dbg.js:3985
j.createComponent @ Component-dbg.js:1340
Excelupload @ Excelupload.js?eval:11
eval @ FPMHelper.ts:48
(anonymous) @ ui5loader-dbg.js:2107
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2106
Oe @ ui5loader-dbg.js:1858
r @ ui5loader-dbg.js:2099
eval @ FPMHelper.ts:45
actionWrapper @ FPMHelper.ts:15
y @ ExpressionParser-dbg.js:412
(anonymous) @ ExpressionParser-dbg.js:735
l @ ExpressionParser-dbg.js:915
l.getExternalValue @ CompositeBinding-dbg.js:316
u @ EventHandlerResolver-dbg.js:349
eval @ EventHandlerResolver-dbg.js:173
r.fireEvent @ EventProvider-dbg.js:241
y.fireEvent @ Element-dbg.js:683
(anonymous) @ ManagedObjectMetadata-dbg.js:826
S.ontap @ library-preload.js?eval:206
y._handleEvent @ Element-dbg.js:352
N._handleEvent @ UIArea-dbg.js:1054
dispatch @ jquery-dbg.js:5430
c @ jquery-mobile-custom-dbg.js:1907
d @ jquery-mobile-custom-dbg.js:2030
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
trigger @ jquery-dbg.js:8823
(anonymous) @ jquery-dbg.js:8901
each @ jquery-dbg.js:385
each @ jquery-dbg.js:207
trigger @ jquery-dbg.js:8900
j @ jquery-mobile-custom-dbg.js:1455
L @ jquery-mobile-custom-dbg.js:1465
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
ui5loader-dbg.js:1156 Uncaught (in promise) ModuleError: failed to load 'cc/spreadsheetimporter/v1_0_0/Component.js' from sesmassuploadapp.sesappwithexcel/webapp/thirdparty/customcontrol/spreadsheetimporter/v1_0_0/Component.js: 404
    at we (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:6645)
    at ge.failWith (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:4709)
    at qe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:13312)
    at Oe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:15314)
    at Object.r [as require] (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:18696)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36487
    at new Promise (<anonymous>)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36454
Caused by: XHRLoadError: 404
    at n (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:9251)
    at XMLHttpRequest.<anonymous> (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:9504)
    at je (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:9596)
    at qe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:13287)
    at Oe (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:15314)
    at Object.r [as require] (https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:10:18696)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36487
    at new Promise (<anonymous>)
    at https://ui5.sap.com/1.120.13/resources/sap-ui-core.js:446:36454Caused by: XHRLoadError: 404
    at n (ui5loader-dbg.js:1369:12)
    at XMLHttpRequest.<anonymous> (ui5loader-dbg.js:1382:21)
    at je (ui5loader-dbg.js:1392:8)
    at qe (ui5loader-dbg.js:1650:5)
    at Oe (ui5loader-dbg.js:1843:20)
    at Object.r [as require] (ui5loader-dbg.js:2099:4)
    at Component-dbg.js:3632:13
    at new Promise (<anonymous>)
    at Component-dbg.js:3630:12
we @ ui5loader-dbg.js:1156
failWith @ ui5loader-dbg.js:934
qe @ ui5loader-dbg.js:1655
Oe @ ui5loader-dbg.js:1843
r @ ui5loader-dbg.js:2099
(anonymous) @ Component-dbg.js:3632
(anonymous) @ Component-dbg.js:3630
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2116
Oe @ ui5loader-dbg.js:1856
r @ ui5loader-dbg.js:2099
(anonymous) @ Component-dbg.js:3632
(anonymous) @ Component-dbg.js:3630
Promise.then (async)
V @ Component-dbg.js:3625
z @ Component-dbg.js:2739
j.create @ Component-dbg.js:2455
n @ Component-dbg.js:1507
b.runWithOwner @ ManagedObject-dbg.js:1213
j.runAsOwner @ Component-dbg.js:762
j._createComponent @ Component-dbg.js:1518
j.createComponent @ Component-dbg.js:1338
Excelupload @ Excelupload.js?eval:11
eval @ FPMHelper.ts:48
(anonymous) @ ui5loader-dbg.js:2107
setTimeout (async)
s @ ui5loader-dbg.js:59
(anonymous) @ ui5loader-dbg.js:2106
Oe @ ui5loader-dbg.js:1858
r @ ui5loader-dbg.js:2099
eval @ FPMHelper.ts:45
actionWrapper @ FPMHelper.ts:15
y @ ExpressionParser-dbg.js:412
(anonymous) @ ExpressionParser-dbg.js:735
l @ ExpressionParser-dbg.js:915
l.getExternalValue @ CompositeBinding-dbg.js:316
u @ EventHandlerResolver-dbg.js:349
eval @ EventHandlerResolver-dbg.js:173
r.fireEvent @ EventProvider-dbg.js:241
y.fireEvent @ Element-dbg.js:683
(anonymous) @ ManagedObjectMetadata-dbg.js:826
S.ontap @ library-preload.js?eval:206
y._handleEvent @ Element-dbg.js:352
N._handleEvent @ UIArea-dbg.js:1054
dispatch @ jquery-dbg.js:5430
c @ jquery-mobile-custom-dbg.js:1907
d @ jquery-mobile-custom-dbg.js:2030
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
trigger @ jquery-dbg.js:8823
(anonymous) @ jquery-dbg.js:8901
each @ jquery-dbg.js:385
each @ jquery-dbg.js:207
trigger @ jquery-dbg.js:8900
j @ jquery-mobile-custom-dbg.js:1455
L @ jquery-mobile-custom-dbg.js:1465
dispatch @ jquery-dbg.js:5430
y.handle @ jquery-dbg.js:5234
```
</details>

### Spreadsheet Component Init

```js
sap.ui.define([
    "sap/m/MessageToast"
], function(MessageToast) {
    'use strict';

    return {
        Excelupload:  async function(oEvent) {
            MessageToast.show("Custom handler invoked.");
            //this._view.setBusyIndicatorDelay(0); 
            //this._view.setBusy(true);
            this.spreadsheetUpload =  await this._view.getController().getAppComponent().createComponent({
                usage: "spreadsheetImporter",
                async: true,
                    componentData: {
                        context: this,
                        debug: true
                    },
                });
            this.spreadsheetUpload.openSpreadsheetUploadDialog();
            //this._view.setBusy(false);
        }
    };
});
```


### Manifest

```json
"componentUsages": {
      "spreadsheetImporter": {
        "name": "cc.spreadsheetimporter.v1_0_0"
      }
    },
    "resourceRoots": {
      "cc.spreadsheetimporter.v1_0_0": "./thirdparty/customcontrol/spreadsheetimporter/v1_0_0"
    },
```

--- a/docs/pages/CentralDeployment.md
+++ b/docs/pages/CentralDeployment.md
@@ -110,6 +110,12 @@ When the component is deployed centrally, you can run the app locally by install
 
 You can install the component as dev dependency in your app to simulate the central deployed component either to ABAP Stack or BTP.
 
+0\. Install [`cds-plugin-ui5`](https://www.npmjs.com/package/cds-plugin-ui5?activeTab=versions) if you use CAP Project
+
+```sh
+npm install cds-plugin-ui5 --save-dev
+```
+
 1\. Install the component as dev dependency (install the component version that is used in your app)
 
 ```sh

--- a/docs/pages/GettingStarted.md
+++ b/docs/pages/GettingStarted.md
@@ -247,7 +247,7 @@ openSpreadsheetUploadDialog: async function (event) {
 ## Deployment of your app
 
 
-### ABAP System Deployment
+### ABAP Stack
 
 #### Component Deployment
 
@@ -341,7 +341,15 @@ customTasks:
       - .*\.js.map
 ```
 
-### BTP Environment Deployment
+### BTP Environment
+
+#### Run with CAP 
+
+If you are using CAP and installing the component as dependency **to your UI5 App**, you have to use [`cds-plugin-ui5`](https://www.npmjs.com/package/cds-plugin-ui5?activeTab=versions), so the UI5 Tooling is loading the installed component.
+
+```sh
+npm install cds-plugin-ui5 --save-dev
+```
 
 #### Config your `ui5-task-zipper` in your deployment yaml file
 

